### DISCUSSION TITLE
A icon in the lower left corner

### DISCUSSION THREAD

**Discussion by RONG-GUI** [pyqtgraph/pyqtgraph#2229] [Q&A] [answered]
How to hide the a icon in the lower left corner?
![image](https://user-images.githubusercontent.com/61581769/158563398-510e263d-8877-4430-9e0e-c6af7514bd78.png)

**Comment by danielhrisca** [comment, marked as answer]
You can use this method https://github.com/pyqtgraph/pyqtgraph/blob/8f7d50cc14d19ff74dcc2166de4ba3a27055b0a3/pyqtgraph/graphicsItems/PlotItem/PlotItem.py#L1286

**Reply by RONG-GUI** [reply]
Thank you for your guidance
